pkg/kube/krt/conformance_test.go

```diff
@@ -135,11 +143,20 @@ func TestConformance(t *testing.T) {
 		}
 		runConformance[Named](t, rig)
 	})
+	t.Run("files", func(t *testing.T) {
+		t.Skip("Not implemented")
+		col := krt.NewFileCollection[Named](krt.WithStop(test.NewStop(t)))
+		rig := &fileRig{
+			FileCollection: col,
+			rootPath:       t.TempDir(),
+		}
+		runConformance[Named](t, rig)
+	})
 }
 
 func runConformance[T any](t *testing.T, collection Rig[T]) {
 	stop := test.NewStop(t)
-	// Collection should start empty...
+	// FileCollection should start empty...
```

Comment:
Is this a find/replace error? I don't see how this is specific to FileCollections